rgw/v2/tests/s3_swift/reusable.py

```diff
@@ -1075,3 +1079,25 @@ def put_bucket_lifecycle(bucket, rgw_conn, rgw_conn2, life_cycle_rule):
         raise TestExecError("bucket life cycle retrieved")
     lc_data = json.loads(utils.exec_shell_cmd("radosgw-admin lc list"))
     log.info(f"lc data is {lc_data}")
+
+
+def get_radoslist():
+    """
+    get radolist of all buckets
+    """
+    cmd = "radosgw-admin bucket radoslist | grep -i ERROR"
+    _, err = subprocess.getstatusoutput(cmd)
+    if err:
+        log.error(f"ERROR in radoslist command! {err}")
+        get_bucket_stats()
+
+
+def get_bucket_stats():
```

Comment:
It's good to return the result so that  it can be used at the places where it is needed

Reply:
sure, added return value